### PR TITLE
docs: emojis for readme

### PR DESCRIPTION
**Pull request by krshkodes**
# Summary

Now with more and more projects using emojis, and other graphical means to make the readme look more attractive, and beautiful, I have added emojis to the headers..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,21 @@
 # Apache DevLake(Incubating)
 
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat&logo=github&color=2370ff&labelColor=454545)](http://makeapullrequest.com)
-![badge](https://github.com/apache/incubator-devlake/actions/workflows/test.yml/badge.svg)
+![Unit Test Status](https://github.com/apache/incubator-devlake/actions/workflows/test.yml/badge.svg)
```
Comment (krshkodes) [inline README.md:9]: I have a question, do we have something we could link to here?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inefficient

Comment (warren830) [reply README.md:9]: do  you mean the summary part?

**Comment by likyh** [comment]
That's interesting. 😁

File: README.md (hunk 1 above)
Comment (krshkodes) [reply README.md:9]: I mean, like we have linked other things, is there any possible URL we could like Unit Test Status to?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by warren830** [comment]
Well, nice work, this makes the repo more vivid

File: README.md (hunk 1 above)
Comment (warren830) [reply README.md:9]: > I mean, like we have linked other things, is there any possible URL we could like Unit Test Status to?
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** ❓ I have a question

Ah, thanks. Actually, we don't have anything to link here. I think maybe we should replace it by [Contributors](https://img.shields.io/github/contributors/apache/echarts?color=3ba272)](https://github.com/apache/incubator-devlake/graphs/contributors)

**Review by warren830 — APPROVED** [review]
LGTM

**Comment by warren830** [comment]
@krshkodes Hi, would you like to join us on slack? We'd like to issue a certificate for you to honor your contribution

**Comment by krshkodes** [comment]
Thank you @warren830! I'm already on the slack! :D

**Pull request merged**